netbox/utilities/request.py

```diff
@@ -17,7 +17,7 @@ def get_client_ip(request, additional_headers=()):
     )
     for header in HTTP_HEADERS:
         if header in request.META:
-            client_ip = request.META[header].split(',')[0]
+            client_ip = request.META[header].split(',')[0].partition(':')[0]
```

Comment:
Just curious, any reason to use `partition()` instead of `split()`?

Reply:
Both `partition()` and `split()` can be used interchangeably. `partition()` is generally used when we need to split a string at the first occurrence of a separator and we are sure there will be only one separator or are only interested in the first occurrence.